Running CI for #30677 - do not merge

metamben opened the pull request:
Running CI for external contribution #30677.

lbrdnk committed:
Unify semicolon and comment handling in nested native queries

lbrdnk committed:
Use different comment and semicolon removal approach

lbrdnk committed:
Apply suggestions from code review

Co-authored-by: metamben <103100869+metamben@users.noreply.github.com>

lbrdnk committed:
Update base and effective type verification for oracle

metamben committed:
Merge branch 'master' into 29168-ending-with-comment

lbrdnk committed:
Avoid running test with snowflake and bigquery

lbrdnk commented:
@metamben, snowflake and bigquery require special handling with regards to table name generation. I think it would be good idea to test those separately. I've disabled testing them in general sql qp test (new commit in original pr). I don't have access to bigquery or snowflake instance at the moment and I will try to add tests for those drivers tomorrow.

metamben committed:
Merge branch 'master' into 29168-ending-with-comment

lbrdnk committed:
Update `breakout-results` and test

1. In `breakout-results` case driver statement was redundant.
Original code had field name and types hardcoded. During further
examination, I've realized name and types are available thanks
to the use of `qp.test/col`, so are just "unhardcoded" :)

2. `card-id-native-source-queries-test` is altered, so instead of
hardcoding native sub-queries, roughly equivalent* mbql query is
compiled and its native form is used. This way I can hopefully avoid
problems with extraction of cloud databases' table names for testing
purposes.

* By roughly equivalent I mean instead of using * for selecting
fields, mbql is compiled selecting fields explicitly. Also limit is added
in some cases, but local testing showed no problems with that.

lbrdnk committed:
Merge remote-tracking branch 'upstream/master' into 29168-ending-with-comment

metamben committed:
Merge branch 'master' into 29168-ending-with-comment

metamben commented:
Original PR has been merged.
